src/top.rs

```diff
@@ -297,7 +297,7 @@ impl Link {
         dst: SocketAddr,
         message: Protocol,
     ) {
-        tracing::trace!(target: TRACING_TARGET, ?src, ?dst, protocol = %message, "Send");
+        tracing::trace!(target: TRACING_TARGET, ?src, ?dst, protocol = %message, "Send (2)");
```

Comment:
Why "Send (2)" here?

Reply:
Oh, that should have been removed, that was only there for debugging purposes.